canvas.gypi

```diff
@@ -0,0 +1,43 @@
+{
+    'variables':
+    {
+        'has_FBDev%': 'true',
```

Comment:
Should this really always be true? I'm not sure if FreeBSD always has it, for example.